docs/csharp/programming-guide/enumeration-types.md

```diff
@@ -36,77 +29,80 @@ translation.priority.ht:
-  
- To determine whether a specific flag is set, use a bitwise `AND` operation, as shown in the following example:  
-  
- [!code-cs[csProgGuideEnums#7](../../csharp/programming-guide/codesnippet/CSharp/enumeration-types_6.cs)]  
-  
- For more information about what to consider when you define enumeration types with the <xref:System.FlagsAttribute?displayProperty=fullName> attribute, see <xref:System.Enum?displayProperty=fullName>.  
-  
-## Using the System.Enum Methods to Discover and Manipulate Enum Values  
- All enums are instances of the <xref:System.Enum?displayProperty=fullName> type. You cannot derive new classes from <xref:System.Enum?displayProperty=fullName>, but you can use its methods to discover information about and manipulate values in an enum instance.  
-  
- [!code-cs[csProgGuideEnums#5](../../csharp/programming-guide/codesnippet/CSharp/enumeration-types_7.cs)]  
-  
- For more information, see <xref:System.Enum?displayProperty=fullName>.  
-  
- You can also create a new method for an enum by using an extension method. For more information, see [How to: Create a New Method for an Enumeration](../../csharp/programming-guide/classes-and-structs/how-to-create-a-new-method-for-an-enumeration.md).  
-
-## See Also  
+> It's possible to assign any arbitrary integer value to `meetingDay`. For example, this line of code does not produce an error: `meetingDay = (Day) 42`. However, you should not do this because the implicit expectation is that an enum variable will only hold one of the values defined by the enum. To assign an arbitrary value to a variable of an enumeration type is to introduce a high risk for errors.
+
+ You can assign any values to the elements in the enumerator list of an enumeration type, and you can also use computed values:
+
+[!code-csharp[csProgGuideEnums#3](../../../samples/snippets/csharp/VS_Snippets_VBCSharp/csProgGuideEnums/CS/Enums.cs#3)]
+
+## Enumeration types as bit flags
+
+ You can use an enumeration type to define bit flags, which enables an instance of the enumeration type to store any combination of the values that are defined in the enumerator list. (Of course, some combinations may not be meaningful or allowed in your program code.)
+
+ You create a bit flags enum by applying the <xref:System.FlagsAttribute?displayProperty=fullName> attribute and defining the values appropriately so that `AND`, `OR`, `NOT` and `XOR` bitwise operations can be performed on them. In a bit flags enum, include a named constant with a value of zero that means "no flags are set." Do not give a flag a value of zero if it does not mean "no flags are set".
+
+ In the following example, another version of the `Day` enum, which is named `Days`, is defined. `Days` has the `Flags` attribute and each value is assigned the next greater power of 2. This enables you to create a `Days` variable whose value is `Days.Tuesday` and `Days.Thursday`.
```

Comment:
attribute and --> attribute, and
`Days.Tuesday` and `Days.Thursday` --> It's unclear what the "and" means here, and confusing because it's not a Boolean Boolean. Perhaps `Days.Tuesday | Days.Thursday`?